### PR TITLE
:lipstick: Create a styleguid on storybook

### PR DESCRIPTION
**Pull request by welschoff**
Screenshot 2021-11-09 at 12 21 29

### CODE REVIEW

File: src/app/globals.css [hunk 1]
```diff
@@ -1,3 +1,43 @@
...
+
+button {
+  border-radius: 15px;
+  border: none;
+  background-color: var(--color-secondary);
+  font-family: var(--font-family);
+  padding: 0.5rem;
+}
+
+/* .box {
```
Comment (riittagerhards) [inline src/app/globals.css:36]: Add a comment or delete :)

File: src/app/globals.css [hunk 2]
```diff
@@ -1,3 +1,43 @@
...
   margin: 0;
 }
+
+button {
+  border-radius: 15px;
+  border: none;
+  background-color: var(--color-secondary);
+  font-family: var(--font-family);
+  padding: 0.5rem;
+}
```
Comment (AnkePfei2fer) [inline src/app/globals.css:34]: You could also put these into your Styleguide.tsx file

**Review by AnkePfei2fer — COMMENTED** [review]
lgtm

**Review by riittagerhards — COMMENTED** [review]
good job! lgtm!

**Pull request merged**